src/SplunkExporter.js

```diff
@@ -0,0 +1,119 @@
+/*
+Copyright 2020 Splunk Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import {
+  toZipkinSpan, 
+  statusCodeTagName, 
+  statusDescriptionTagName,
+} from '@opentelemetry/exporter-zipkin/build/src/transform.js';
+import {ExportResultCode} from '@opentelemetry/core';
+import {limitLen} from './utils';
+import { Resource } from '@opentelemetry/resources';
+
+const MAX_VALUE_LIMIT = 4096;
+const SPAN_RATE_LIMIT_PERIOD = 30000; // millis, sweep to clear out span counts
+const MAX_SPANS_PER_PERIOD_PER_COMPONENT = 100;
+const SERVICE_NAME = 'browser';
+
+const SPAN_TO_ATTRIBUTES = (span) => span.attributes;
```

Comment:
Idk about this `(span) => attributes` input-output mismatch

1. Either missing out on chance to also apply this to other data on spans (exceptions _should_ be an event on some span, but that might be future), or to filter out spans (eg. return null)
2. Or how about `(attributes, span) => attributes`, since most will need `span.attributes` anyway and then can `(attrs) => ({...attrs, whatever})` for most cases, or use span from 2nd argument if they need it for some reason

Reply:
good point